day7/will/index.css

```diff
@@ -0,0 +1,682 @@
+body {
+  margin: 0px;
+  font-family: "TTnorms", sans-serif;
+  height: 100vh;
+  width: 100%;
+}
+
+@font-face {
+  font-family: "TTnorms";
+  src: url("/ttnorms/bold.woff2") format("woff2");
+  src: url("/ttnorms/light.woff2") format("woff2");
+  src: url("/ttnorms/medium.woff2") format("woff2");
+  src: url("/ttnorms/regular.woff2") format("woff2");
+  src: url("/ttnorms/bold.woff") format("woff");
+  src: url("/ttnorms/light.woff") format("woff");
+  src: url("/ttnorms/medium.woff") format("woff");
+  src: url("/ttnorms/regular.woff") format("woff");
+}
+
+@media only screen and (max-width: 767px) {
+  .tabletview-wrapper {
+    display: none;
+  }
+
+  .laptop-wrapper {
+    display: none;
+  }
+
+  .mobileview-wrapper {
+    position: relative;
+    background: linear-gradient(0deg, #6666ff 0, #002b80 100%);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100%;
+    overflow: hidden;
+  }
+
+  .mobileview-img-phone {
+    width: 100%;
+    margin-top: 50px;
+    margin-bottom: 0px;
+  }
+
+  .mobileview-container {
+    position: absolute;
+    top: 50px;
+    text-align: center;
+  }
+
+  .mobileview-container img {
+    width: 25%;
+    z-index: -1;
```

Comment:
img can be index of 0, -1 is below surface

Reply:
adjusted